contracts/calculator/calculator.algo.ts

```diff
@@ -0,0 +1,48 @@
+import { arc4, err } from "@algorandfoundation/algorand-typescript";
+import type { uint64 } from "@algorandfoundation/algorand-typescript";
+
+// eslint-disable-next-line no-unused-vars
+export class Calculator extends arc4.Contract {
```

Comment:
I think it'd be good if we tried to reduce the usage of `arc4` as much as possible (ie. just use `Contract` instead of `arc4.Contract`). TEALScript has no explicit mention of ARC4/ABI anywhere in the API and obviously ARC4 is new to TypeScript developers. The more code we can write without having to use it the better and finding areas where it's required is during the migration process is a good way to figure out what needs to be worked on

Reply:
It has been updated to use just `Contract`.